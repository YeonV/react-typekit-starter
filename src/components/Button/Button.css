.BasicButton {
    background-color: #800000;
}