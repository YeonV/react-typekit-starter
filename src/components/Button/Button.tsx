import React from 'react'
import './Button.css';

interface Props {
    color: string
}

const Button = (props: Props) => {
    return (
        <div className={'BasicButton'} style={{backgroundColor: props.color}}>
            this will be a button
        </div>
    )
}

export default Button
