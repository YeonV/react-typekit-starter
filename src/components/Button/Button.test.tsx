import React from 'react';
import { shallow } from 'enzyme';
import Button from './Button';


test('renders the component', () => {
  const component = shallow(<Button color="#FFF000" />);
  expect(component).toMatchSnapshot();
});