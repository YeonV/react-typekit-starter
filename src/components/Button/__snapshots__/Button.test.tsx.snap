// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div
  className="BasicButton"
  style={
    Object {
      "backgroundColor": "#FFF000",
    }
  }
>
  this will be a button
</div>
`;
